### PR TITLE
Fungible Token V1

### PR DESCRIPTION
**Pull request by emilypi**

### CODE REVIEW

File: pact/coin-contract/coin.pact [hunk 1]
```diff
@@ -50,18 +48,46 @@
...
-
   (defcap ACCOUNT_GUARD (account)
     "Lookup and enforce guards associated with an account"
     (with-read coin-table account { "guard" := g }
       (enforce-guard g)))
 
-  (defcap GOVERNANCE ()
-    (enforce false "Enforce non-upgradeability except in the case of a hard fork"))
+  (defcap DEBIT (sender:string amount:decimal)
+    "Debit is protected by SENDER guard."
```
Comment (slpopejoy) [inline pact/coin-contract/coin.pact]: Comment is inaccurate

File: pact/coin-contract/coin.pact [hunk 2]
```diff
@@ -50,18 +48,46 @@
     "Magic capability constraining genesis transactions"
     true)
 
-  (defcap FUND_TX ()
-    "Magic capability to execute gas purchases and redemptions"
-    true)
-
   (defcap ACCOUNT_GUARD (account)
```
Comment (slpopejoy) [inline pact/coin-contract/coin.pact]: It looks like this is unused now

File: pact/coin-contract/coin.repl [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
+(load "fungible-v1.pact")
 (load "coin.pact")
 
 (commit-tx)
 (begin-tx)
 
-;; HUGE GLARING TODO - pending FV fixes in Pact
-;(verify 'coin)
+; HUGE GLARING TODO - pending FV fixes in Pact
```
Comment (slpopejoy) [inline pact/coin-contract/coin.repl]: `verify` is uncommented, do we need this comment

Comment (emilypi) [reply pact/coin-contract/coin.repl]: no - it's just warnings atm

File: pact/coin-contract/coin.pact (hunk 2 above)
Comment (emilypi) [reply pact/coin-contract/coin.pact]: removed

File: pact/coin-contract/coin.pact (hunk 1 above)
Comment (emilypi) [reply pact/coin-contract/coin.pact]: fixed

**Pull request merged**